src/renderers/webgl/WebGLTextures.js

```diff
@@ -753,14 +775,14 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 	// Render targets
 
 	// Setup storage for target texture and bind it to correct framebuffer
-	function setupFrameBufferTexture( framebuffer, renderTarget, attachment, textureTarget ) {
+	function setupFrameBufferTexture( framebuffer, renderTarget, texture, attachment, textureTarget ) {
 
 		var glFormat = utils.convert( renderTarget.texture.format );
 		var glType = utils.convert( renderTarget.texture.type );
```

Comment:
This should be looking at `texture.type` (same with `format` above), right? As it is all settings on the `texture[]` targets are being ignored.

Reply:
Good catch, thanks!